index.js

```diff
@@ -2193,12 +2221,12 @@ if (!window.console) {
      * var cb = ClearBlade.Messaging({}, callback);
      * cb.unsubscribe("ClearBlade/is awesome!",{"onSuccess":function(){console.log("we unsubscribe");});
      */
-    messaging.unsubscribe = function(topic, options) {
+    messaging.unsubscribe = function (topic, options) {
       var conf = {};
-      conf["invocationContext"] = options["invocationContext"] || {};
-      conf["onSuccess"] = options["onSuccess"] || function() {}; //null;
-      conf["onFailure"] = options["onFailure"] || function() {}; //null;
-      conf["timeout"] = options["timeout"] || 60;
+      conf['invocationContext'] = options['invocationContext'] || {};
+      conf['onSuccess'] = options['onSuccess'] || function () {}; //null;
+      conf['onFailure'] = options['onFailure'] || function () {}; //null;
+      conf['timeout'] = options['timeout'] || 60;
       this.client.unsubscribe(topic, conf);
```

Comment:
do we need to remove the callback from `messageCallbacks` on unsub?

Reply:
Yea, that would be good